CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 #### 0.2.1 (Next)
 
-* Your contribution here.
+* [#10](https://github.com/dblock/gem-licenses/pull/10): Rename class in order to avoid conflict with Rubygems 2.5 - [@caiosba](https://github.com/caiosba)
```

Comment:
This is missing a period at the end. Also explain what was actually done, like "Renamed `Gem::Licenses` to `Gem::GemLicenses`...."